libraries/src/Image/Image.php

```diff
@@ -1141,11 +1141,7 @@ protected function sanitizeWidth($width, $height)
      */
     public function destroy()
     {
-        if ($this->isLoaded()) {
-            return imagedestroy($this->getHandle());
-        }
-
-        return false;
+        return $this->isLoaded();
```

Comment:
```suggestion
        $this->handle = null;

        return true;
```

this function is also to free resources, I would expect if there are no references to the property anymore it will be freed and garbage collected later.